### PR TITLE
[examples] update ember example

### PR DESCRIPTION
**Pull request by trek**
Show Ember dev welcome message, even in production environment.

### CODE REVIEW

**Comment by styfle** [comment]
Closing in favor of https://github.com/vercel/vercel/pull/10347

File: examples/ember/ember-cli-build.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
 module.exports = function (defaults) {
   const app = new EmberApp(defaults, {
-    // Add options here
+    'ember-welcome-page': {
```
Comment (cb1kenobi) [inline examples/ember/ember-cli-build.js:7]: Out of curiosity, is this documented?

Comment (trek) [reply examples/ember/ember-cli-build.js:7]: Configuring Ember apps? Or the config options for this demo component?

Comment (cb1kenobi) [reply examples/ember/ember-cli-build.js:7]: This config option.

Comment (trek) [reply examples/ember/ember-cli-build.js:7]: Not that I know of, no. The component is meant to be thrown away when you develop your app.

**Pull request closed without merging**